lib/tutorials/index.js

```diff
@@ -95,9 +100,26 @@ module.exports = function makePlugin(dir) {
 
             let tutorialTags = new Set();  // set of tags related to tutorials
 
+            // get all tutorial tags
+            for (const filepath in files) {
+                if (filepath.indexOf('.DS_Store') >= 0) continue;
```

Comment:
Minor, style wise I prefer to have continue on the next line as it's a separate logic statement and easier to add breakpoints when debugging

Reply:
Thanks, hadn't thought about that